full-stack-serverless/02-getting-started/amplify-react-app/src/App.js

```diff
@@ -0,0 +1,61 @@
+// React hooks
+import React, { useState, useEffect } from 'react';
+
+// API category from AWS Amplify
+import { API } from 'aws-amplify';
+
+import './App.css';
+
+function App() {
+	// Create `coins` variable and set to empty array
+	const [coins, updateCoins] = useState([]);
+	// Create additional state to hold user input for `limit` and `start` properties
+	const [input, updateInput] = useState({ limit: 5, start: 0 });
+
+	function updateInputValues(type, value) {
+		updateInput({ ...input, [type]: value });
+	}
```

Comment:
```suggestion
	function updateInputValues(type, value) {
		if (value) {
			updateInput({ ...input, [type]: value });
		}
	}
```

Maybe not the greatest user experience, but if, as the "user", 
1. I set the `limit` and `start` properties to some values
2. Click `Fetch Coins`
3. Remove/reset the properties
4. Click `Fetch Coins` again, the `""` values are passed in, and we fall back on the LiteCoin API's defaults. 🤔